test/vfd_swmr_common.c

```diff
@@ -202,7 +254,37 @@ await_signal(hid_t fid)
 
     dbgf(1, "waiting for signal\n");
 
+#if defined(__APPLE__) && defined(__MACH__)
```

Comment:
Why the `__MACH__`?

Reply:
That's something I pulled from Stackoverflow, IIRC. Things like that are used to differentiate between MacOS and iOS. Eventually, I'll change this so its conditional on sigtimedwait (I think FreeBSD also lacks sigtimedwait, and this hack won't help on that platform), but right now I just want to get it to compile on Macs.

Reply:
Now I test on the presence of sigtimedwait, so this is obsolete.